gpcontrib/arenadata_toolkit/sql/upgrade_test.sql

```diff
@@ -0,0 +1,134 @@
+-- Change log level to disable notice messages from PL/pgSQL and dropped objects
+-- from "DROP SCHEMA arenadata_toolkit CASCADE;"
+SET client_min_messages=WARNING;
+
+-- We have only first and last versions of arenadata_toolkit and scripts to update
```

Comment:
I don't think so, instead we have [all intermediate versions](https://github.com/arenadata/gpdb/blob/e73000417c345a3e6ec913c17fb9eb04d8e6f8ee/gpcontrib/arenadata_toolkit/Makefile#L16) of update scripts, so, you may simplify upgrade tests.